spark/src/test/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcherTest.java

```diff
@@ -968,10 +968,14 @@ private String constructExpectedSparkSubmitParameterString(
         + "  --conf"
         + " spark.datasource.flint.customAWSCredentialsProvider=com.amazonaws.emr.AssumeRoleAWSCredentialsProvider"
         + "  --conf"
-        + " spark.sql.extensions=org.opensearch.flint.spark.FlintSparkExtensions,org.opensearch.flint.spark.FlintPPLSparkExtensions"
+        + " spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.opensearch.flint.spark.FlintSparkExtensions,org.opensearch.flint.spark.FlintPPLSparkExtensions"
         + "  --conf"
         + " spark.hadoop.hive.metastore.client.factory.class=com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
         + "  --conf"
+        + " spark.sql.catalog.spark_catalog=org.apache.iceberg.spark.SparkSessionCatalog"
```

Comment:
spark_catalog is the default one right, what is the impact of doing these?

Reply:
Yes `spark_catalog` is default catalog. I didn't see any impact in testing, it looks like the default catalog behavior is correctly fallen back to if the table is not an Iceberg table. This is exactly the config that Athena is using as well. 

Added more details to testing section.

Reply:
we have the below configuration where we have a dummy catalog with the datasource name.
 --conf spark.sql.catalog.my_glue=org.opensearch.sql.FlintDelegatingSessionCatalog 
 Few questions.
* I am actually trying to understand on how this is going to work existing hiive table. Wont the catalog require every query made would be an iceberg table. Are these covered in spark repo integ tests?

Reply:
No Iceberg falls back to default Spark catalog if it's not an Iceberg table. When creating tables you still have to specify using iceberg to create an iceberg table. It will still use the default implementation.

I couldn't find it in public documentation, but in internal documentation there is this line: `org.apache.iceberg.spark.SparkSessionCatalog adds support for Iceberg tables to Spark's built-in catalog, and delegates to the built-in catalog for non-Iceberg tables`